src/clientiface.h

```diff
@@ -396,6 +396,16 @@ class RemoteClient
 	std::map<v3s16, float> m_blocks_sending;
 
 	/*
+		Blocks that have been modified since last sending them.
+		These blocks will not be marked as sent, even if the
+		client reports it has received them to account for blocks
+		that are being modified while on the line.
+
+		Key is position, value is dummy
```

Comment:
No need to replicate the `value is dummy` for `m_blocks_sent`. It is a mistake left over from commit 6a1670dbc31cc0e44178bbd9ad34ff0d5981a060 (the commit replaced map with set but didnt update the comment).